### DISCUSSION TITLE
[FE] Migrate to Remix or Next.js

### DISCUSSION THREAD

**Discussion by leo-diehl** [DudaLipinski/Boardie#84] [Ideas]
https://react.dev/learn/start-a-new-react-project
https://nextjs.org/docs/advanced-features/turbopack
https://remix.run

**Comment by DudaLipinski** [comment]
After migrating the app to Next.js, we arrived at some topics that Next.js could not be the best solution for us, like authentication.

We'll maintain this branch and think a lit bit more about it before finishing the migration.

What is missing to migrate:
- Update react query
- Create a structure to handle when the user is authenticated without rendering the page

Branch: https://github.com/DudaLipinski/Boardie/tree/82-fe-migrate-to-nextjs
